security/user/user.go

```diff
@@ -39,3 +39,74 @@ func (su SecurityUser) Fetch(id string, options *types.Options) (types.User, err
 
 	return user, nil
 }
+
+/*
+  Executes a search on Users according to filters.
+*/
+func (su SecurityUser) Search(filters interface{}, options *types.Options) (types.KuzzleSearchUsersResult, error) {
+	ch := make(chan types.KuzzleResponse)
```

Comment:
Hey guys, someone needs to reconfigure the tab / spaces in the editor here. This file has a completely different indentation from the rest.